lib/workers/repository/update/branch/auto-replace.spec.ts

```diff
@@ -178,6 +178,67 @@ describe('workers/repository/update/branch/auto-replace', () => {
       );
     });
 
+    it('updates with autoReplaceNewString without Digest and when PinDigests is false no digest is added', async () => {
```

Comment:
You can put tests in a table in Jest with functions like `describe.each(table)(name, fn, timeout)`) [^docs], it looks like this:

```ts
describe.each([
  [1, 1, 2],
  [1, 2, 3],
  [2, 1, 3],
])('.add(%i, %i)', (a, b, expected) => {
  test(`returns ${expected}`, () => {
    expect(a + b).toBe(expected);
  });

  test(`returned value not be greater than ${expected}`, () => {
    expect(a + b).not.toBeGreaterThan(expected);
  });

  test(`returned value not be less than ${expected}`, () => {
    expect(a + b).not.toBeLessThan(expected);
  });
});
```

Maybe a table layout is easier to understand than the current `autoReplaceNewString` tests?

I'll let the maintainers decide if you should put the tests in such a table. 😉 You don't have to do anything right now. 😉 

[^docs]: https://jestjs.io/docs/api#describeeachtablename-fn-timeout